plugin-tester/src/index.js

```diff
@@ -15,27 +15,43 @@ program.command('init <name>')
         template.createTemplate(name)
     })
 
-const readConfig = (pluginPath, options) => {
+const resolveConfigPath = config => {
+    return path.resolve(config)
+}
+
+const readJson = path => {
+    return JSON.parse(fs.readFileSync(path))
+}
+
+const createConfigReader = (pluginPath, options) => () => {
     const configPath = path.join(pluginPath, 'config.json')
     if (options.config) {
-        return require(path.resolve(options.config))
+        return readJson(resolveConfigPath(options.config))
```

Comment:
`require` caches behind the scenes, hence why we use `decache` for the loading of plugins. It's semantically more correct to load this as jsonFile rather than require.